src/ui/MLFacet/MLFacet.ts

```diff
@@ -248,6 +247,28 @@ export class MLFacet extends Component {
     this.handleFacetValuesChanged();
   }
 
+  /**
+   * Requests an additional amount of values equal to the [`number of values`]{@link MLFacet.options.numberOfValues}.
+   *
+   * Does trigger a query automatically.
+   */
+  public showMoreValues(): void {
+    this.ensureDom();
+    this.logger.info('Show more values', (this.mLFacetQueryController.numberOfValuesToRequest += this.options.numberOfValues));
+    this.triggerNewQuery();
+  }
+
+  /**
+   * Reduce the amount of values down to the [`number of values`]{@link MLFacet.options.numberOfValues}.
```

Comment:
Reduces the number of displayed facet values down to [`numberOfValues`]{@link MLFacet.options.numberOfValues}.